client/my-sites/site-settings/form-general.jsx

```diff
@@ -467,6 +487,30 @@ class SiteSettingsFormGeneral extends Component {
 					</form>
 				</Card>
 
+				{ ! siteIsJetpack &&
+					<div>
+						<SectionHeader label={ translate( 'Net Neutrality' ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Neutral' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).